From 471fa7c76c6db51f42710c4401bf23aa132a3d79 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Fri, 04 Jun 2021 08:11:30 +0300
Subject: [PATCH] [SW-0] preboot: align preboot size to master

Align Preboot size in greco to what is used in master (66K) to
accommodate static protection implementations.

Change-Id: I9a1d0d7f6484d46787c3b81522a038fed94e887f
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/greco/common.h b/greco/common.h
index 85a7f3c..22694f3 100644
--- a/greco/common.h
+++ b/greco/common.h
@@ -11,7 +11,7 @@
 #include <bitops.h>
 
 #define PPBOOT_SIZE			0x5000		/* 20KB */
-#define PREBOOT_SIZE			0x10000		/* 64KB */
+#define PREBOOT_SIZE			0x10800		/* 66KB */
 
 #define SRAM_SIZE			0x8000000	/* 128MB */
 #define HL_SP_SRAM_SIZE			0x10000
